src/components/frame/v5/pages/UserCryptoToFiatPage/partials/FiatTransfersTable/FiatTransferDescriptionRow.tsx

```diff
@@ -0,0 +1,23 @@
+import clsx from 'clsx';
+import React, { type PropsWithChildren, type FC } from 'react';
+
+interface FiatTransferDescriptionRowProps {
+  title: string;
+  loading: boolean;
+}
+
+export const FiatTransferDescriptionRow: FC<
+  PropsWithChildren<FiatTransferDescriptionRowProps>
+> = ({ title, children, loading }) => {
+  const textClassName = 'font-normal text-sm flex items-center';
```

Comment:
I guess we can either wrap this in `tw` or just smash it down into the className directly